src/drivers/boards/px4fmu-v2/board_config.h

```diff
@@ -134,6 +147,9 @@ __BEGIN_DECLS
 #define PX4_SPIDEV_EXT_ACCEL_MAG	PX4_SPIDEV_EXT2
 #define PX4_SPIDEV_EXT_GYRO		PX4_SPIDEV_EXT3
 
+#define	PX4_SPIDEV_EXT_MAG		PX4_SPIDEV_EXT_BARO
+#define	PX4_SPIDEV_EXT_ACCEL_GYRO	PX4_SPIDEV_EXT_ACCEL_MAG
```

Comment:
This is all a bit confusing if you have to read it in some weeks/months time. Could you add some comments about what is what?